proto/decentraland/kernel/apis/players.proto

```diff
@@ -23,7 +23,10 @@ message GetPlayersInSceneRequest {}
 message GetConnectedPlayersRequest {}
 
 service PlayersService {
+    // @deprecated, only available for SDK6 compatibility
     rpc GetPlayerData(GetPlayerDataRequest) returns (PlayersGetUserDataResponse) {}
+    // @deprecated, only available for SDK6 compatibility
     rpc GetPlayersInScene(GetPlayersInSceneRequest) returns (PlayerListResponse) {}
+    // @deprecated, only available for SDK6 compatibility
```

Comment:
// Returns a list of all the ids of players who are currently connected to the same server and grouped together
// NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7